### PR TITLE
shellcheck: SC2269: This variable is assigned to itself

### PR DESCRIPTION
**Pull request by alan-barzilay**
https://github.com/koalaman/shellcheck/wiki/SC2269

### CODE REVIEW

File: src/pomodoro.sh [hunk 1]
```diff
@@ -250,9 +250,8 @@ function calculate_missing_time()
     m)
       time_value=$((60 * "$time_value"))
       ;;
-    s)
-      time_value="$time_value"
-      ;;
+    s) ;;
+
```
Comment (joaoseckler) [inline src/pomodoro.sh:254]: Maybe the `s` case could be totally left out. But maybe this way is more readable.

**Review by joaoseckler — APPROVED** [review]

File: src/pomodoro.sh (hunk 1 above)
Comment (rodrigosiqueira) [reply src/pomodoro.sh:254]: Yeah... let's remove this `s)`

**Review by rodrigosiqueira — CHANGES_REQUESTED** [review]
@alan-barzilay let me know when you have a new version

**Comment by alan-barzilay** [comment]
@rodrigosiqueira done

**Comment by rodrigosiqueira** [comment]
Merged,
Please, take care with your commit message.
Thanks

**Pull request closed without merging**